src/main/java/eu/europeana/metis/sandbox/service/util/XsltUrlUpdateServiceImpl.java

```diff
@@ -31,27 +32,30 @@ public void updateXslt(String defaultXsltUrl) {
     HttpRequest httpRequest;
     try {
       httpRequest = HttpRequest.newBuilder()
-          .GET()
-          .uri(URI.create(defaultXsltUrl))
-          .build();
+                               .GET()
+                               .uri(URI.create(defaultXsltUrl))
+                               .build();
 
       xsltStream = httpClient.send(httpRequest, BodyHandlers.ofInputStream()).body();
 
     } catch (IOException | InterruptedException e) {
       // try to load resource as file
       xsltStream = getClass().getClassLoader().getResourceAsStream(defaultXsltUrl);
+      Thread.currentThread().interrupt();
```

Comment:
More a doubt? I this to wait for the stream resource to load?

Reply:
This is when ```InterruptedException``` occurs. Sonar reported to include this code line.  This will release the resources of the thread and do not swallow the exception. 
https://stackoverflow.com/questions/4906799/why-invoke-thread-currentthread-interrupt-in-a-catch-interruptexception-block
see the comment of Java Concurrency in practice.